src/System.Linq.Expressions/src/System/Dynamic/DynamicObject.cs

```diff
@@ -219,6 +219,7 @@ public virtual System.Collections.Generic.IEnumerable<string> GetDynamicMemberNa
         {
             return Array.Empty<string>();
```

Comment:
This might be a good time to just remove this and use `Array.Empty<string>()` directly. It seems to just serve to replace the now-idiomatic call to that method for no benefit (though presumably it was once its own separately cached empty array, before we were using `Empty()` everywhere).